### ISSUE TITLE
NPE IN 3.2.0 on log statement

### ISSUE THREAD

**Issue by ctoestreich** [bertramdev/grails-asset-pipeline#343]
AP 3.2.0 in deployed war

```
Caused by: groovy.lang.MissingPropertyException: No such property: log for class: asset.pipeline.AssetPipelineGrailsPlugin
Possible solutions: scm
	at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.unwrap(ScriptBytecodeAdapter.java:53)
	at org.codehaus.groovy.runtime.callsite.PogoGetPropertySite.getProperty(PogoGetPropertySite.java:52)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGroovyObjectGetProperty(AbstractCallSite.java:307)
	at asset.pipeline.AssetPipelineGrailsPlugin.doWithApplicationContext(AssetPipelineGrailsPlugin.groovy:63)
	at org.grails.plugins.DefaultGrailsPlugin.doWithApplicationContext(DefaultGrailsPlugin.java:524)
	at org.grails.plugins.AbstractGrailsPluginManager.doPostProcessing(AbstractGrailsPluginManager.java:229)
	at grails.boot.config.GrailsApplicationPostProcessor.onApplicationEvent(GrailsApplicationPostProcessor.groovy:229)
	at grails.boot.config.GrailsApplicationPostProcessor.onApplicationEvent(GrailsApplicationPostProcessor.groovy)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:163)
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:136)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:381)
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:335)
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:855)
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.finishRefresh(EmbeddedWebApplicationContext.java:140)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:541)
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:118)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:764)
	at org.springframework.boot.SpringApplication.doRun(SpringApplication.java:357)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:305)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:54)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:353)
	at grails.boot.GrailsApp.run(GrailsApp.groovy:342)
	at grails.boot.GrailsApp$run.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:133)
	at bsb.fingerhut.web.Application.main(Application.groovy:13)
```

**Comment by davydotcom** [comment]
fixed in 3.2.1

**Comment by davydotcom** [comment]
3.2.2 will have a further fix that just keeps it from even trying to register exploded plugins as a war
